src/assets/src/components/queueManager.tsx

```diff
@@ -404,6 +404,13 @@ export function QueueManagerPage(props: PageProps<QueueManagerPageParams>) {
     const queueWebSocketError = useQueueWebSocket(queueIdParsed, setQueueChecked);
     const [visibleMeetingDialog, setVisibleMeetingDialog] = useState(undefined as Meeting | undefined);
 
+    const [myUser, setMyUser] = useState(undefined as MyUser | undefined);
+    const userWebSocketError = useUserWebSocket(props.user!.id, (u) => setMyUser(u as MyUser));
+
+    if (myUser && queue) {
+        checkBackendAuth(myUser, queue);
+    }
+
```

Comment:
I think this is okay for now, but I wonder if adding another WebSocket connection to this page is a somewhat heavy solution. Did you explore passing `authorized_backends` through `globals`?

Reply:
I had a few things in mind when I went this way...
* If you have multiple tabs open, your zoom auth wouldn't be noticed until you refresh the page, unless you're getting a fresh User via API or websocket.
* We might eventually want to move away from the current redirect and instead pop up a modal without leaving the SPA. The modal would have the same authorize link, except it would open in a new tab, and the tab would close when the flow is complete. This would handle users with multiple open tabs or queues with multiple auth-required backends better, because you could display a modal that has multiple authorize links, and it can vanish automatically when all of the backends are authorized.
* I've been thinking that, at a high level, it would be convenient to have a global User websocket always open no matter which page you're on. In which case, it would make sense to source authorized_backends from there. I mimicked that in a way that involved less refactoring.

In this case, if I replaced this websocket with a call to the User API, it would be functionally the same, except in an extreme edge case:  A non-Zoom auth'd user has two tabs open, `/manage/0` and `/manage/1`; `0` toggles Zoom to be an `allowed_backend`; the user is prompted and completes the auth; then `1` toggles Zoom to be an `allowed_backend`. If using websockets, `1` correctly updates itself with Zoom as a `authorized_backend`. If using the API, `1` prompts for Zoom auth again. (Not likely to ever happen).

IOW, given 3 choices that were only narrowly different, I picked the one that looked closest to where things might eventually end up.

Reply:
I think I follow all of that; it does seem like we'd benefit from a WS in the multiple tabs situation. I guess I was a concerned about adding yet another way to get user info that could get out of sync, but `globals` doesn't give you much that would change anyway. I was also thinking you only need the update once, so why maintain a WS connection after that, but it may not matter much performance-wise if nothing changes.